src/cloudant/replicator.py

```diff
@@ -163,14 +163,11 @@ def update_state():
             Retrieves the replication state.
             """
             try:
-                repl_doc = self.database[repl_id]
-                repl_doc.fetch()
-                state = repl_doc.get('_replication_state')
+                arepl_doc = self.database[repl_id]
+                arepl_doc.fetch()
+                return arepl_doc, arepl_doc.get('_replication_state')
             except KeyError:
-                repl_doc = None
-                state = None
-
-            return repl_doc, state
+                return None, None
```

Comment:
Pycharm and I found the scope a little unnerving as repl_doc is used in the outer parent function (review the larger context).  Changing the inner function variable to arepl_doc removes potential confusion on if you intend to manipulate the outer variable or create a new one.  I will undo the edit if desired.